lib/ex_unit_cluster.ex

```diff
@@ -44,4 +44,38 @@ defmodule ExUnitCluster do
       ExUnitCluster.call(unquote(cluster), unquote(node), unquote(module_name), :run, [])
     end
   end
+
+  @doc """
+  Execute multiline code blocks on a specific node,
+  inheriting environment variables from the caller.
```

Comment:
Environment variables is ambiguous, maybe something like "inheriting variables from the caller scope."

Reply:
Or rather, "capturing"